versions/run.ver

```diff
@@ -61,4 +61,5 @@ export rtofs_ver=v2.3
 export sref_ver=v7.1
 export st4_ver=v4.1
 export urma_ver=v2.10
+export bufr_ver=12.0.0
```

Comment:
@BinbinZhou-NOAA These are in alphabetical order. Can you please move `export bufr_ver=12.0.0` right after `export bacio_ver=2.4.1`?

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA